app/controllers/oauth_controller.rb

```diff
@@ -21,6 +21,12 @@ def failure
     render_forbidden params.require(:message)
   end
 
+  def development_log_in_as
```

Comment:
Any reason to use term `log_in` in here (and also at `log_in_as` and `ENABLE_DEVELOPMENT_ADMIN_LOG_IN`) instead of `login`? I'm not native speaker, so I have no idea what's the difference. But in current codebase there is `login` used constantly.

Reply:
https://grammarist.com/spelling/log-in-login/

Reply:
TIL thanks